sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredentialBuilder.java

```diff
@@ -110,7 +145,7 @@ public DefaultAzureCredential build() {
     }
 
     private ArrayDeque<TokenCredential> getCredentialsChain() {
-        ArrayDeque<TokenCredential> output = new ArrayDeque<>(4);
+        ArrayDeque<TokenCredential> output = new ArrayDeque<>(5);
```

Comment:
Just FYI both this PR and the VsCodeCredential PR increment this to 5. After you merge one you'll need to make sure to update the other PR.